index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <img src="/images/Welcome.png" alt="welcome button at the top left">
```

Comment:
- [ ] you should delete the image and try to implement the social media icons with `<a>` and also should be wrapped with `<ul>` each one with `<li>` as in the Figma design in order to match it 
- [ ] try to use flexbox to align them in order to match the design

Reply:
bruh can you please give me some hints on how I can use flexbox to align them, I have tried but failed

Reply:
@PrinceBadru you can use `display: flex;` and they will align Horizontally and use `gap` to give some space between the icons or margin whatever you prefer